linera-rpc/src/message.rs

```diff
@@ -50,13 +55,33 @@ impl RpcMessage {
             | Error(_)
             | ChainInfoResponse(_)
             | VersionInfoQuery
-            | VersionInfoResponse(_) => {
+            | VersionInfoResponse(_)
+            | DownloadBlob(_)
+            | DownloadBlobResponse(_) => {
                 return None;
             }
         };
 
         Some(chain_id)
     }
+
+    pub fn is_local_message(&self) -> bool {
```

Comment:
Would be good to add some documentation. I guess "local" here means that the proxy can answer it directly, without forwarding the message to a server process?